### PR TITLE
Feat/card hover

### PR DESCRIPTION
**Pull request by zoe-klein**

### CODE REVIEW

File: TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    } else {
+      const temp: string[] = [];
+
+      attributes.forEach((attribute) => {
+        temp.push(attribute.attributeName, ' ');
+      });
+
+      setProperties(temp);
+    }
+  }, [attributes, cardType]);
```
Comment (DeannaShort) [inline TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx:81]: This line causes the code to break if there is a card without attributes, can be seen when viewing pokemon. Would suggest adding a line like this.
```suggestion
    if (attributes.length === 0) return;
    if (attributes[0].attributeName.includes(cardType.name)) {
      setProperties([cardType.name]);
    } else {
      const temp: string[] = [];

      attributes.forEach((attribute) => {
        temp.push(attribute.attributeName, ' ');
      });

      setProperties(temp);
    }
  }, [attributes, cardType]);
```

File: TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx [hunk 2]
```diff
@@ -68,27 +67,19 @@ export function CardImageDisplay({ imageUrl }: CardImageDisplayProps) {
...
-                <AspectRatio ratio={63 / 88}>
-                  <img className={classes.modalImageStyling} src={imageUrl} />
-                </AspectRatio>
-              </Grid.Col>
-            </Grid>
+        <Modal.Content>
+          <Modal.CloseButton iconSize={25} size={25} />
+
+          <Modal.Body>
+            <Image radius={17} src={imageUrl} security="yamomma" />
```
Comment (DeannaShort) [inline TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]:
```suggestion
            <Image radius={17} src={imageUrl}  />
```

File: TcgPocket/App/src/components/filterMenu/CategoryAndOptions.tsx [hunk 3]
```diff
@@ -62,8 +62,6 @@ export function CategoryAndOptions({
   }, [appliedFilters, filteredOptions]);
 
   const handleFilters = (option: CardTypeGetDto) => {
-    dispatch(setSearchTextInventory(''));
-    dispatch(setCurrentPage(1));
```
Comment (DeannaShort) [inline TcgPocket/App/src/components/filterMenu/CategoryAndOptions.tsx:66]: This code is necessary for the inventory page. When selecting a filter, you need to return back to page 1 and the search text needs cleared.

**Review by DeannaShort — CHANGES_REQUESTED** [review]

File: TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx (hunk 2 above)
Comment (xalluna) [reply TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]: this is vital to the integrity of the application

Comment (DeannaShort) [reply TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]: 👎

File: TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx (hunk 1 above)
Comment (xalluna) [reply TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx:81]: I'll do you one better, this should all be in a `useMemo`, no `useEffect` or `useState`.

```
const properties = useMemo(() => {
  if (attributes.length === 0) return [];
  const tempAttributes = attributes.map(x => x.attributeName);
  attributes[0].attributeName.includes(cardType.name) {
    tempAttributes.push(cardType.name);
  }
  return tempAttributes;
},[attributes, cardType]);
```

**Review by xalluna — APPROVED** [review]
make dee happy but otherwise looks good to me

File: TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx (hunk 2 above)
Comment (zoe-klein) [reply TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]: Do I keep this or not? 🥲

Comment (xalluna) [reply TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]: idk it looks like dee is really upset about. probably not the worst idea to get rid of it 🥲

Comment (DeannaShort) [reply TcgPocket/App/src/components/cardDisplay/modules/CardImageDisplay.tsx:82]: Lol I am not. I was just meme-ing

File: TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx (hunk 1 above)
Comment (zoe-klein) [reply TcgPocket/App/src/pages/deckBuilder/modules/HoverInventoryCard.tsx:81]: I like the useMemo :0

**Review by DeannaShort — APPROVED** [review]

**Pull request merged**